src/lib/helpers.js

```diff
@@ -47,3 +47,9 @@ export const isType: IsType = type => val => {
       return typeof val === type;
   }
 };
+
+/**
+ * Flattens two arrays
+ */
+type Flatten = (Array<any>, Array<any>) => Array<any>;
+export const flatten: Flatten = (acc, vs) => [...acc, ...vs];
```

Comment:
1. We should add tests for this method in `helpers.spec.js`.

2. In a previous comment (https://github.com/times/schema-validator/pull/17#discussion_r124508407) I suggested we make this a more generic method (like `const flatten = arr => arr.reduce(...)`) rather than one that will only work as the first argument to a `reduce`, as it is now. I still think we should do that because this signature is weird on its own.

3. On reflection I wonder whether this file might be named poorly. It completely makes sense for `flatten` to live in a `helpers` file, but the other methods in there are more like typecheckers than general helpers. Should we break them out into `typecheck.js` or similar? Thoughts welcome.